cluster/kube-down.sh

```diff
@@ -35,7 +35,7 @@ verify-prereqs
 echo "... calling verify-kube-binaries" >&2
 verify-kube-binaries
 echo "... calling kube-down" >&2
-if [[ $PRESET_INSTANCES_ENABLED == $TRUE ]]; then
+if [[ ${PRESET_INSTANCES_ENABLED:-} == $TRUE ]]; then
```

Comment:
I suggested to use default value = false.

Reply:
Done